andela/matterhorn-cfh
# #151753514 leaderboard of past games

#### What does this PR do?
It allows authenticated users to view leader board after each games
#### Description of Task to be completed?
NA
#### How should this be manually tested?
After playing a game, user can click go to their dashboard, click leaderboard button and see the board.
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
As a registered user I want to view a scoreboard/leaderboard of previously played games so that I can reference the overall winner of the different games played and scores of other participants of the game.
#### Screenshots (if appropriate)
<img width="1372" alt="screen shot 2017-10-13 at 11 08 00 pm" src="https://user-images.githubusercontent.com/13118321/31568407-1d1114f8-b06c-11e7-8817-dda30926426d.png">

#### Questions:
